opencsp/common/lib/csp/SolarField.py

```diff
@@ -318,7 +318,7 @@ def draw(
 
         # Draw Origin
         if solar_field_style.draw_origin:
-            view.draw_single_Pxyz(origin)
+            Pxyz(origin).draw_point(view)

```

Comment:
Why is Pxyz(...) necessary here? It looks like `origin` is already type Pxyz. There are also several other cases of this I've seen but not pointed out each one. 
If `origin` is in facet already a Pxyz, this is not necessarily wrong, just redundant and possibly confusing. You're call.

Reply:
This was because origin was a Vyxz and needed to be a Pxyz and I wanted to draw it with the draw_point() method. Now that the method is in Vyxz class I don't need this conversion anymore.